src/scripts/dfnshelper.js

```diff
@@ -0,0 +1,103 @@
+import * as locale from 'date-fns/locale'
+define(["globalize"], function (globalize) {
+    "use strict";
+
+    function getLocale()
+    {
+        switch (globalize.getCurrentLocale()) {
+            case 'ar':

```

Comment:
I would rather see this is a dictionary, rather than this giant switch

Reply:
Hmm I've done it this way because I thought it would be more readable than the globalize implementation (which uses a dict). I'm open to whatever it's decided, I guess it's a matter of discussing this.

Reply:
dict storing looks more compact without losing the readability for my taste. Though by no means count this as a blocker, if others are fine with the PR they can go ahead and merge :)